app/styles/app.scss

```diff
@@ -3,3 +3,19 @@ $color: red;
 h1 {
   color: $color;
 }
+
+// stylelint-disable-next-line selector-pseudo-class-no-unknown
```

Comment:
Instead of disabling it here, maybe we should `:global` in the list of exceptions for this rule since we are using css-modules in this project. What do you think?